consensus-types/types/state.go

```diff
@@ -58,6 +58,9 @@ type BeaconState struct {
 	// Slashing
 	Slashings     []math.Gwei
 	TotalSlashing math.Gwei
+
+	// Deposits
+	DepositsRoot common.Root
```

Comment:
This is not needed if we stick to `Eth1Data`

Reply:
Dropped